Gemfile

```diff
@@ -3,4 +3,6 @@ gem 'guard-sass'
 gem 'guard-coffeescript'
 gem 'html2haml'
 gem 'guard-haml'
+gem 'guard-shell'
 gem 'rb-readline'
+gem 'therubyracer'
```

Comment:
It would probably be better to use https://github.com/sstephenson/execjs instead of therubyracer. I believe that's more of the standard now.